src/core/raster/qgsrasterprojector.cpp

```diff
@@ -832,17 +835,19 @@ QgsRasterBlock *QgsRasterProjector::block( int bandNo, QgsRectangle  const &exte
       char *destBits = outputBlock->bits( destIndex );
       if ( !srcBits )
       {
-        QgsDebugMsg( QString( "Cannot get input block data: row = %1 col = %2" ).arg( i ).arg( j ) );
+        //  QgsDebugMsg( QString( "Cannot get input block data: row = %1 col = %2" ).arg( i ).arg( j ) );
```

Comment:
What's the reason to remove these debug messages?

Are they printed in normal situations (aka noise because expected) or no longer should be triggered (aka should go to message log)?